doc/usecases/gamification/map_collaboration.md

```diff
@@ -0,0 +1,30 @@
+# Use Case: Map Collaboration
+
+## Summary
+
+- **Scope:** Map Collaboration
+- **Level:** User Goal
+- **Actors:** Map Creator, Collaborator
+- **Brief:** collaborators can plan on the map of a map creator.
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  - A map creator has created a map and at least one other collaborator is registered in Nextcloud.
```

Comment:
We register via keycloak and everyone automatically is available in Nextcloud, too.

```suggestion
  - A map creator has created a map and at least one other collaborator is registered.
```